README.md

```diff
@@ -84,7 +84,7 @@ Apache Airflow is tested with:
 | PostgreSQL   | 9.6, 10, 11, 12, 13       | 9.6, 10, 11, 12, 13      | 9.6, 10, 11, 12, 13        |
 | MySQL        | 5.7, 8                    | 5.7, 8                   | 5.6, 5.7                   |
 | SQLite       | 3.15.0+                   | 3.15.0+                  | 3.15.0+                    |
-| Kubernetes   | 1.16.9, 1.17.5, 1.18.6    | 1.16.9, 1.17.5, 1.18.6   | 1.16.9, 1.17.5, 1.18.6     |
+| Kubernetes   | 1.20, 1.19, 1.18          | 1.20, 1.19, 1.18         | 1.18, 1.17, 1.16           |
```

Comment:
What does this means for Airflow 2.0.x users that have K8s 1.16?
is this considered a breaking change from Airflow side?

Reply:
Very good point @eladkal, but I don't think there is any compatibility problem/breaking change. We never had any policy for K8S, so technically we are not breaking any 'promises'. However this is important to clarify it.

As mentioned in the commit/description Kubernetes 1.16 and 1.17 are already not supported (1.17 support ended 31 Jan 2021). 
By this change we are following here the very exact process we agreed to follow for python versions.

This change is just implementation of the same approach https://github.com/apache/airflow#support-for-python-versions - I am happy to propose (and ask for lazy consensus) explicitly in the devlist (will do it later today).

Also, we are not telling it will "not work" with those versions, only that we do not test it with those older versions. This means that at some point in time it might break, but it's not very likely as those APIs are rather stable.